### PR TITLE
Development

### PR DESCRIPTION
**Pull request by kekey1**

### CODE REVIEW

File: pulse/broker/src/main/java/gov/ca/emsa/pulse/broker/dao/impl/AlternateCareFacilityDAOImpl.java [hunk 1]
```diff
@@ -178,11 +190,21 @@ private AlternateCareFacilityEntity getEntityById(Long id) {
...
+		query.setParameter("identifier", identifier);
+		return query.getResultList();
+	}
+	
 	private List<AlternateCareFacilityEntity> getEntityByName(String name) {
 		Query query = entityManager.createQuery( "SELECT DISTINCT a "
 				+ "FROM AlternateCareFacilityEntity a "
 				+ "LEFT OUTER JOIN FETCH a.lines lines "
-				+ "where (a.name LIKE :name) "
+				+ "where (a.name LIKE :name)"
```
Comment (andlar) [inline pulse/broker/src/main/java/gov/ca/emsa/pulse/broker/dao/impl/AlternateCareFacilityDAOImpl.java:207]: Should this be `a.identifier LIKE :identifier`?

**Review by brianlindsy — APPROVED** [review]

File: pulse/broker/src/test/java/gov/ca/emsa/pulse/broker/manager/AcfManagerTest.java [hunk 2]
```diff
@@ -87,7 +87,7 @@ public void createAcfWithAddress() throws SQLException  {
 		String zip = "21227";
 		
 		AlternateCareFacilityDTO dto = new AlternateCareFacilityDTO();
-		dto.setName(name);
+		dto.setIdentifier(name);
```
Comment (andlar) [inline pulse/broker/src/test/java/gov/ca/emsa/pulse/broker/manager/AcfManagerTest.java]: `name` or `identifier`?

File: pulse/broker/src/test/java/gov/ca/emsa/pulse/broker/manager/AcfManagerTest.java [hunk 3]
```diff
@@ -102,7 +102,7 @@ public void createAcfWithAddress() throws SQLException  {
 		Assert.assertNotNull(dto);
 		Assert.assertNotNull(dto.getId());
 		Assert.assertTrue(dto.getId().longValue() > 0);
-		Assert.assertEquals(name, dto.getName());
+		Assert.assertEquals(name, dto.getIdentifier());
```
Comment (andlar) [inline pulse/broker/src/test/java/gov/ca/emsa/pulse/broker/manager/AcfManagerTest.java]: `name` or `identifier`?

File: pulse/broker/src/test/java/gov/ca/emsa/pulse/broker/manager/AcfManagerTest.java [hunk 4]
```diff
@@ -115,7 +115,7 @@ public void createAcfWithAddress() throws SQLException  {
...
```
Comment (andlar) [inline pulse/broker/src/test/java/gov/ca/emsa/pulse/broker/manager/AcfManagerTest.java]: `name` or `identifier`?

File: pulse/broker/src/test/java/gov/ca/emsa/pulse/broker/manager/AcfManagerTest.java [hunk 5]
```diff
@@ -128,20 +128,55 @@ public void createAcfWithAddress() throws SQLException  {
 	@Test
 	@Transactional
 	@Rollback(true)
+	public void updateAcfFriendlyName() throws SQLException  {
+		String name = "ACF 1";
```
Comment (andlar) [inline pulse/broker/src/test/java/gov/ca/emsa/pulse/broker/manager/AcfManagerTest.java]: `name` or `identifier`?

File: pulse/broker/src/test/java/gov/ca/emsa/pulse/broker/manager/AcfManagerTest.java [hunk 6]
```diff
@@ -128,20 +128,55 @@ public void createAcfWithAddress() throws SQLException  {
...
+		String phoneNumber = "4105551000";
+		
+		AlternateCareFacilityDTO dto = new AlternateCareFacilityDTO();
+		dto.setIdentifier(name);
```
Comment (andlar) [inline pulse/broker/src/test/java/gov/ca/emsa/pulse/broker/manager/AcfManagerTest.java]: `name` or `identifier`?

File: pulse/broker/src/test/java/gov/ca/emsa/pulse/broker/manager/AcfManagerTest.java [hunk 7]
```diff
@@ -128,20 +128,55 @@ public void createAcfWithAddress() throws SQLException  {
...
+		dto.setPhoneNumber(phoneNumber);
+		dto = acfManager.create(dto);
+		Long acfId = dto.getId();
+		
+		Assert.assertNotNull(dto);
+		Assert.assertNotNull(acfId);
+		Assert.assertTrue(acfId.longValue() > 0);
+		Assert.assertEquals(name, dto.getIdentifier());
```
Comment (andlar) [inline pulse/broker/src/test/java/gov/ca/emsa/pulse/broker/manager/AcfManagerTest.java]: `name` or `identifier`?

File: pulse/broker/src/test/java/gov/ca/emsa/pulse/broker/manager/AcfManagerTest.java [hunk 8]
```diff
@@ -128,20 +128,55 @@ public void createAcfWithAddress() throws SQLException  {
...
+		Assert.assertNull(dto.getName());
+		Assert.assertEquals(phoneNumber, dto.getPhoneNumber());
+		
+		String friendlyName = "M&T Bank Stadium";
+		dto.setName(friendlyName);
```
Comment (andlar) [inline pulse/broker/src/test/java/gov/ca/emsa/pulse/broker/manager/AcfManagerTest.java]: `friendlyName`?

File: pulse/broker/src/test/java/gov/ca/emsa/pulse/broker/manager/AcfManagerTest.java [hunk 9]
```diff
@@ -128,20 +128,55 @@ public void createAcfWithAddress() throws SQLException  {
...
+	@Transactional
+	@Rollback(true)
 	public void updateAcfPhoneNumber() throws SQLException  {
 		String name = "ACF 1";
 		String phoneNumber = "4105551000";
 		String updatedPhoneNumber = "3015551000";
 		
 		AlternateCareFacilityDTO dto = new AlternateCareFacilityDTO();
-		dto.setName(name);
+		dto.setIdentifier(name);
```
Comment (andlar) [inline pulse/broker/src/test/java/gov/ca/emsa/pulse/broker/manager/AcfManagerTest.java]: `name` or `identifier`?

File: pulse/broker/src/test/java/gov/ca/emsa/pulse/broker/manager/AcfManagerTest.java [hunk 10]
```diff
@@ -128,20 +128,55 @@ public void createAcfWithAddress() throws SQLException  {
...
 		dto.setPhoneNumber(phoneNumber);
 		dto = acfManager.create(dto);
 		
 		Assert.assertNotNull(dto);
 		Assert.assertNotNull(dto.getId());
 		Assert.assertTrue(dto.getId().longValue() > 0);
-		Assert.assertEquals(name, dto.getName());
+		Assert.assertEquals(name, dto.getIdentifier());
```
Comment (andlar) [inline pulse/broker/src/test/java/gov/ca/emsa/pulse/broker/manager/AcfManagerTest.java]: `name` or `identifier`?

There are a few more...

**Review by andlar — CHANGES_REQUESTED** [review]

File: pulse/broker/src/main/java/gov/ca/emsa/pulse/broker/dao/impl/AlternateCareFacilityDAOImpl.java (hunk 1 above)
Comment (kekey1) [reply pulse/broker/src/main/java/gov/ca/emsa/pulse/broker/dao/impl/AlternateCareFacilityDAOImpl.java:207]: No.. that's the method above this one. This method is for a lookup by friendly name, which is now called "name".

**Review by andlar — APPROVED** [review]

**Pull request merged**